packages/cli/package.json

```diff
@@ -38,7 +38,9 @@
     "@rollup/plugin-json": "^4.0.2",
     "@rollup/plugin-node-resolve": "^8.1.0",
     "@rollup/plugin-yaml": "^2.1.1",
-    "@spotify/eslint-config": "^7.0.1",
+    "@spotify/eslint-config-base": "^8.0.0",
+    "@spotify/eslint-config-react": "^8.0.0",
+    "@spotify/eslint-config-typescript": "^8.0.0",
```

Comment:
We weren't using `@spotify/eslint-config` directly, so splitting this up lets us manage the other dependencies independently